### DISCUSSION TITLE
Download and run the install script: ( not working )

### DISCUSSION THREAD

**Discussion by abbarsumairi** [amidaware/tacticalrmm#1661] [General]
Hi there 

script installation not working 

wget https://raw.githubusercontent.com/amidaware/tacticalrmm/master/install.sh
chmod +x install.sh
./install.sh

error code ( failed: Connection timed out.
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8001::154|:443... failed: Connection timed out.
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8003::154|:443...   ) 

please help

**Comment by wh1te909** [comment]
github isn't down. fix your internet lol

**Reply by abbarsumairi** [reply]
I'm sure internet is connected 100%, but i don't what is the issue

**Reply by dinger1986** [reply]
Can you browse to it?

**Comment by abbarsumairi** [comment]
> Can you browse to it?

Yes. i can brows youtube and google.com

**Reply by dinger1986** [reply]
No to the script

**Reply by abbarsumairi** [reply]
i can't download it to browse it

**Reply by dinger1986** [reply]
So if you try and open it?

https://raw.githubusercontent.com/amidaware/tacticalrmm/master/install.sh

Can you browse GitHub?

**Reply by abbarsumairi** [reply]
Yes i can browse githup no issue

**Reply by dinger1986** [reply]
So what happens when you open the link?

**Comment by abbarsumairi** [comment]
> No to the script


wget https://raw.githubusercontent.com/amidaware/tacticalrmm/master/install.sh
chmod +x install.sh
./install.sh
